src/file-3.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:29367/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:29367/project/issues?id=sample&branch=feature2&issues=AWDuuHrJ3EyD_61MTRWM&open=AWDuuHrJ3EyD_61MTRWM)

[ISSUE_KEY]: AWDuuHrJ3EyD_61MTRWM